Source/WebInspectorUI/UserInterface/Views/SearchSidebarPanel.js

```diff
@@ -482,3 +496,5 @@ WI.SearchSidebarPanel = class SearchSidebarPanel extends WI.NavigationSidebarPan
         this.focusSearchField();
     }
 };
+
+WI.SearchSidebarPanel.MinimumQueryLengthForSearchOnFocus = 2;
```

Comment:
rather than change the shape of `WI.SearchSidebarPanel` for something that's only used once, i think we should just make this a `const` right before where it's used in `focusSearchField`

also, i kinda feel like maybe `3` feels more reasonable as a minimum for auto-searching